web/themes/custom/votegov/templates/block/block-content--search.html.twig

```diff
@@ -23,12 +28,11 @@
   <input class="usa-input" data-test="search-box" id="search-field-en-small" type="search" name="query"
          autocomplete="off"
          placeholder="{{ content.field_search_placeholder | field_value }} {{ affiliated_search != '1' ? '(in English)' | t }}">
-
-  <button class="usa-button" type="submit">
-    <img
-      src="{{ uswds_img_path }}/usa-icons-bg/search--white.svg"
-      class="usa-search__submit-icon"
-      alt="{{ 'Search Vote.gov' | t }}"
-    />
-  </button>
+  <div class="arrow-icon" role="button" type="submit" tabIndex="0">
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" fill="none">
+      <path d="M0 0L16 8L0 16L4.29127 8L0 0Z" fill="#11385B"/>
+    </svg>
```

Comment:
Move this svg into a file located at `votegov/img/svg/arrow.svg` Then use TWIG `{{ source() }}` function to add it to the template.